generators/app/templates/resource/tasks/config/webpack.config.js

```diff
@@ -0,0 +1,142 @@
+const webpack = require('webpack'),
+  path = require('path'),
+  ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+module.exports = {
+  // Where to start
+  entry: {
+    TagInput: path.resolve(__dirname, '../../src/index.js')
+  },
+
+  // Where to output
+  output: {
+    path: path.resolve(__dirname, '../../dist'),
+    filename: '[name].js',
+    libraryTarget: 'commonjs2'
+  },
+
+  externals: {
+    classnames: true,
+    react: 'React',
+    'react-dom': true,
+    'ship-components-outsideclick': true,
```

Comment:
Let's not assume a component will use this

Reply:
Sure, commented out with a helper note